### ISSUE TITLE
Retire this repo in favor of php/php-src issues?

### ISSUE THREAD

**Issue by nikic** [php/php-tasks#34]
I originally created this repo so we could make use of github issues in a limited fashion while bugs.php.net is still our primary bug tracker. Nowadays, we use github issues on php/php-src, so I think this repo has outlived its usefulness. Should we move all the issues to php-src and archive this repo?

**Comment by Girgias** [comment]
Ah right, I had forgotten why this repo came into existence.

One reason to keep this repo it that we have a lot of issues on the php-src repo that could drown out some of this more "Meta" issues, but then we could always pin them.

**Comment by cmb69** [comment]
I'm also in favor to retire this repository.

**Comment by jimwins** [comment]
Makes sense to me. My PRs to fix the text on this repo and link to it from CONTRIBUTING.md were just because I didn’t even know this existed until it was mentioned on another issue.

Maybe adding a label for the “Meta” issues would be enough. “Good first issue” should be carried over to the php-src issue tracker, too. Or get used more if it’s already there and just not being used.

**Comment by cmb69** [comment]
> “Good first issue” should be carried over to the php-src issue tracker, too. Or get used more if it’s already there and just not being used.

Maybe. But we have that for doc-en, and it doesn't appear that it attracts PRs (https://github.com/php/doc-en/labels/good%20first%20issue).
